conf/systemd/packetfence-keepalived.service

```diff
@@ -13,6 +14,7 @@ ExecStartPre=/bin/perl -I/usr/local/pf/lib '-Mpf::services::manager::keepalived'
 ExecStart=/usr/sbin/keepalived -f /usr/local/pf/var/conf/keepalived.conf --pid=/usr/local/pf/var/run/keepalived.pid
 ExecReload=/bin/kill -HUP $MAINPID
 Restart=on-failure
+Slice=packetfence.slice
 
 [Install]
-WantedBy=packetfence-cluster.target
+WantedBy=packetfence.target
```

Comment:
Not sure it will behave correctly in a cluster where it will start before radiusd load balancer is ready

Reply:
It's because i need it in a standalone mode, so i need to change the target.